services/order.py

```diff
@@ -0,0 +1,39 @@
+from django.db import transaction
+from db.models import Order, Ticket, User, MovieSession
+from django.db.models import QuerySet
+from django.utils.dateparse import parse_datetime
+
+
+def create_order(tickets: list, username: str, date: str = None) -> None:
```

Comment:
Indicate a list of what instances you expect to obtain for `tickets` in the type annotation